app/views/api/v1/users/_info.json.jbuilder

```diff
@@ -1,3 +1,4 @@
-json.id       user.id
-json.name     user.full_name
-json.email    user.email
+json.id        user.id
+json.name      user.full_name
+json.email     user.email
+json.avatar_id user.avatar.record.id
```

Comment:
I think you should use `polymorphic_url ` in order to return the path of the blob created